src/tests/ftest/pool/destroy_rebuild.py

```diff
@@ -34,26 +34,35 @@ def test_destroy_while_rebuilding(self):
         targets = self.server_managers[0].get_config_value("targets")
         ranks = self.params.get("rank_to_kill", "/run/testparams/*")
 
-        # Create a pool
-        self.add_pool()
+        # 1.
+        self.log_step("Create a pool")
+        pool = self.get_pool()
 
-        # Verify the pool information before starting rebuild
+        # 2.
+        self.log_step("Verify the pool information before starting rebuild")
         checks = {
             "pi_nnodes": len(self.hostlist_servers),
             "pi_ntargets": len(self.hostlist_servers) * targets,
             "pi_ndisabled": 0,
         }
         self.assertTrue(
-            self.pool.check_pool_info(**checks),
+            pool.check_pool_info(**checks),
             "Invalid pool information detected prior to rebuild")
 
-        # Start rebuild
+        # 3.
+        self.log_step("Start rebuild, system stop")
         self.server_managers[0].stop_ranks(ranks, self.d_log, force=True)
-        self.pool.wait_for_rebuild_to_start()
+        pool.wait_for_rebuild_to_start()
 
-        # Destroy the pool while rebuild is active
-        self.pool.destroy()
+        # 4.
+        self.log_step("Destroy the pool while rebuild is active")
+        pool.destroy()
 
-        self.log.info("Test Passed")
-        self.get_dmg_command().system_start(",".join([str(x) for x in ranks]))
+        # 5.
+        self.log_step("System start")
+        self.get_dmg_command().system_start(list_to_str(value=ranks))
```

Comment:
Looks like we need to import `list_to_str` from `general_utils`, right?

Reply:
Please see the new commit.  Thanks